crates/oxc_parser/src/js/list.rs

```diff
@@ -265,6 +265,7 @@ impl<'a> SeparatedList<'a> for FormalParameterList<'a> {
         let span = p.start_span();
         p.eat_decorators()?;
 
+        // TODO: use parse_modifiers
```

Comment:
Better to not have a TODO in the parser.

Reply:
fixed